src/controllers/entities.js

```diff
@@ -1,17 +1,16 @@
 const HAPIRestAPI = require('@envage/hapi-pg-rest-api')
 const Joi = require('joi')
-const { get } = require('lodash')
 const { version } = require('../../config')
 const { pool } = require('../lib/connectors/db')
-const { groupBy, reduce } = require('lodash')
 const Boom = require('@hapi/boom')
 
 const isIndividual = entity => {
-  return get(entity, 'entity_type', '').toLowerCase() === 'individual'
+  const entityType = entity.entity_type ? entity.entity_type : ''
```

Comment:
We could use nullish coalescing here (ie. the double question mark operator) which does the same thing, ie. "if the thing on the left of the `??` exists then use that; otherwise, use the thing on the right of it"
```suggestion
  const entityType = entity.entity_type ?? ''
```